style.css

```diff
@@ -3,43 +3,87 @@
 * {
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
 }
 
 html {
   font-family: 'Poppins', sans-serif;
 }
 
+ul {
+  list-style: none;
+}
+
 body {
-  position: absolute;
-  background-color: #e5e5e5;
+  position: relative;
+  background-color: #f7f7f9;
+}
+
+.heading-large {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 52px;
+}
+
+.paragraph {
+  color: #344563;
+  font-style: normal;
+  font-weight: 100;
+  font-size: 18px;
+  line-height: 24px;
 }
 
 header {
+  background-color: #fff;
   color: #6070ff;
   display: flex;
   justify-content: space-between;
   align-items: center;
   padding: 1em 7%;
-  background-color: #fff;
 }
 
 .logo > a {
   text-decoration: none;
   color: inherit;
+  left: 25px;
+}
+
+.nav-menu-button {
+  margin-right: 24px;
+  margin-left: auto;
+  cursor: pointer;
+}
+
+.nav-links {
+  display: none;
+  color: #344563;
+  font-weight: 600;
+}
+
+.nav-link {
+  cursor: pointer;
+  padding: 0.625em 0.75em;
+}
+
+.nav-link:nth-child(even) {
+  margin: 0 0.5em;
 }
 
 .headline-main {
   max-height: 800px;
-  min-height: 100vh;
+  min-height: 550px;
   width: auto;
   display: flex;
   justify-content: center;
   align-items: center;
   padding: 0 7%;
-  background-image: url(./assets/header-shape.png);
+  border-radius: 0 0 0 4em;
   background-color: #fff;
+  background-image: url(./assets/header-shape.png);
+  background-size: 100%;
+  background-position-x: center;
```

Comment:
Very nice incorporation of suggestions but I found out that design does not fit to the screen size. I have testing a few thing, and given the suggestion below, that would resolve your issue. But first look at your design:
![image](https://user-images.githubusercontent.com/61361037/221347824-5d73702b-bea9-4bde-a8f8-d619662c63a9.png)

So incorporate the padding, and it will be all set.

.headline-main {
    ....
    ....
    padding-top: Xpx;
    padding-bottom: Xpx;
}